recipe/migrations/ucx116.yaml

```diff
@@ -0,0 +1,8 @@
+__migrator:
+  build_number: 1
+  commit_message: Rebuild for ucx 1.16
+  kind: version
+  migration_number: 1
+migrator_ts: 1714785040.2362041
+ucx:
+- '1.16'
```

Comment:
Looks like the bot dropped the `.0` that we have kept in the past

Fixing in PR: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5837